Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -220,6 +234,14 @@ public extension CarPlayManagerDelegate {
         logUnimplemented(protocolType: CarPlayManagerDelegate.self, level: .debug)
     }
     
+    /**
+     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
+     */
+    func carPlayManager(_ carPlayManager: CarPlayManager, shouldPresentArrivalUIFor waypoint: Waypoint) -> Bool {
```

Comment:
I’m only now noticing that the existing delegate methods by this name would’ve been a good opportunity to use a preposition other than the overused “for”: `carPlayManager(_:shouldPresentArrivalUIFor:)`. 🤷‍♂️